src/test/java/com/github/mc1arke/sonarqube/plugin/scanner/ScannerConfigurationLoaderSensorTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2019 Michael Clarke
```

Comment:
Please update the year in the copyright notice, and I'd suggest you put your name in here rather than mine